doc/users/next_whats_new/fig_draw_without_rendering.rst

```diff
@@ -1,10 +1,10 @@
-Figure now has draw_no_output method
-------------------------------------
+Figure now has draw_without_rendering method
+--------------------------------------------
 
-Rarely, the user will want to trigger a draw without making output to 
-either the screen or a file.  This is useful for determining the final 
+Rarely, the user will want to trigger a draw without making output to
```

Comment:
"Draw without making output" was meant to mirror the name of the method, so maybe change to the more jargon-y "draw without rendering"?

Reply:
Made this and a couple other changes.